docs/multisigs/emergency-brakes.md

```diff
@@ -95,6 +95,12 @@ GateSeal [`0x1ad5cb2955940f998081c1ef5f5f00875431aa90`](https://etherscan.io/add
 
         - pause capability
 
+
+###### Zircuit Token Bridge: [`0x912C7271a6A3622dfb8B218eb46a6122aB046C79`](https://etherscan.io/address/0x912C7271a6A3622dfb8B218eb46a6122aB046C79)
```

Comment:
@TheDZhon, we do not use the (proposed) mark in the emergency-brakes.md file, correct?

Reply:
yeah, see no issue here tbh